ocw_data_parser/utils.py

```diff
@@ -280,3 +285,42 @@ def parse_all(  # pylint: disable=too-many-arguments, too-many-locals
             parser.export_parsed_json(
                 s3_links=s3_links, upload_parsed_json=perform_upload
             )
+
+
+def convert_to_vtt(loaded_json):
+    """
+    Convert a json file with srt captions data to vtt format
+
+    Args:
+        loaded_json (dict): The content of JSON file
+
+    Returns:
+        dict: copy of the json file with the _datafield_file converted to vtt format
+    """
+    if ".vtt" in loaded_json["id"]:
+        return None
+    new_json = dict(loaded_json)
+    new_json["id"] = loaded_json["id"].replace(".srt", ".vtt")
+    new_json["technical_location"] = loaded_json["technical_location"].replace(
+        ".srt", ".vtt"
+    )
+
+    binary_data = get_binary_data(loaded_json)
+    if binary_data is not None:
+        with tempfile.NamedTemporaryFile() as file:
+            file.write(binary_data)
+            file_name = f"{file.name}.vtt"
```

Comment:
Is this the filename where `webvtt` will store its output?

Reply:
yes, it uses the file name of the original file provided by default.